frontend/src/features/mapSideBar/containers/MapSideBarContainer.tsx

```diff
@@ -153,16 +154,22 @@ const MapSideBarContainer: React.FunctionComponent<IMapSideBarContainerProps> =
     nameSpace?: string,
   ) => {
     fetchParcelsDetail(pidOrPin)(dispatch).then(resp => {
-      const matchingParcel: any = resp?.data?.length ? _.first(resp?.data) : undefined;
+      const matchingParcel: (IParcel & ISearchFields) | undefined = resp?.data?.length
+        ? _.first(resp?.data)
+        : undefined;
       if (!!formikRef?.current?.values && !!matchingParcel?.id) {
         const { setValues, values } = formikRef.current;
         matchingParcel.searchPid = getIn(values, withNameSpace('searchPid', nameSpace));
         matchingParcel.searchPin = getIn(values, withNameSpace('searchPin', nameSpace));
         matchingParcel.searchAddress = getIn(values, withNameSpace('seachAddress', nameSpace));
-        matchingParcel.financials = getMergedFinancials([
-          ...matchingParcel.evaluations,
-          ...matchingParcel.fiscals,
-        ]);
+        matchingParcel.evaluations = getMergedFinancials(
```

Comment:
If we load a parcel based on pin we still need all of the extra evaluations/fiscals in case we want to add new evaluations/fiscals.